.vscode/settings.json

```diff
@@ -15,4 +15,11 @@
   "eslint.format.enable": true,
   "editorconfig.generateAuto": false,
   "files.trimTrailingWhitespace": true,
+  "workbench.colorCustomizations": {
```

Comment:
Diff probably should be removed

Reply:
I'd imagine vscode does a recursive join of all config where `~/.vscode` is higher priority than `project/.vscode`. Opinionated things should be local?

Reply:
Not sure why that got in but usually doesn't happen... 
Anyways I amended that commit.